src/errors.ts

```diff
@@ -179,6 +179,16 @@ export class EtcdInvalidAuthTokenError extends Error {}
  */
 export class EtcdPermissionDeniedError extends Error {}
 
+/**
+ * Etcd leader election has no leader
+ */
+export class EtcdNoLeaderError extends Error {}
```

Comment:
Maybe some default error message?

Reply:
Repeat name of this error class? I don't know what can I talk about this error 😣